src/components/top-features.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+
+ const TopFeatures = () => (
+  <section>
+    <div class="cmp-percentage-card-grid">
+      <div class="cmp-percentage-card">
+        <svg class="cmp-percentage-card__icon" width="264" height="264" viewBox="0 0 264 264">
+          <g fill="none" fill-rule="evenodd">
+            <path fill="#DF138C" fill-opacity="0.1" stroke="#DF138C" stroke-width="2" d="M131.9 1C204 1 262.7 53.5 262.7 117.5a72.4 72.4 0 0 1-73 72.3h-25.1a21.5 21.5 0 0 0-21.8 21.8 22 22 0 0 0 5.4 14.3c3.4 4.1 5.5 9.5 5.5 15 0 12.3-9.6 21.8-21.8 21.8a130.9 130.9 0 0 1 0-261.7"/>
```

Comment:
Would you mind seeing if there is a way to set global variables in React to define these colors once and then use them throughout the partials as we would in Sass?